lib/hdk_semantic_indexes/zome_derive/src/lib.rs

```diff
@@ -154,6 +159,7 @@ pub fn index_zome(attribs: TokenStream, input: TokenStream) -> TokenStream {
                 "Remote" => format_ident!("index_{}_{}", record_type_str_attribute, relationship_name),
                 _ => panic!("expected index type of Local or Remote"),
             };
+            let remote_record_time_index_id: String = format!("{}.indexed", related_record_type_str_attribute);
```

Comment:
I think this is the question to my answer above. It'd be nice to move this into a helper function that generated this as it feels duplicate and would just be easy for it go getting misplaced in the code.